pkg/server/routes.go

```diff
@@ -70,6 +70,7 @@ func (srv *RegistrationServer) SetupRoutes(proxyPort string, reg *prometheus.Reg
 		unsecuredV1.GET("/health", healthCheckCtrl.GetHandler) // TODO: move to root (`/`)?
 		unsecuredV1.GET("/authconfig", authConfigCtrl.GetHandler)
 		unsecuredV1.GET("/segment-write-key", analyticsCtrl.GetDevSpacesSegmentWriteKey) //expose the devspaces segment key
+		unsecuredV1.GET("/analytics/segment-write-key", analyticsCtrl.GetSandboxSegmentWriteKey)
```

Comment:
While the change itself is fine, I wonder how difficult would it be to change the devspaces segment key endpoint to something like `/devspaces/segment-write-key`? Because the placing of something specific to devspaces at the root while something relating to the sandbox itself at a lower level seems a bit "surprising". Not a blocker though.

Reply:
It's not difficult to change on our side. But it will break devspaces. We would need to collaborate with them on it and do a migration. Let's leave it out of scope for this change.